config/client_config/me/system/config/templates/features/brokers/brokers.yml

```diff
@@ -25,9 +25,9 @@ registry:
         is_enabled: true
         item: "https://coverme.inquisiqlms.com/"
       - key: :allow_edit_broker_email
-        is_enabled: true
+        is_enabled: <%= ENV['ALLOW_EDIT_BROKER_EMAIL_IS_ENABLED'] || true %>
```

Comment:
@jayreddy519 do we want to default to true? idk

Reply:
> @jayreddy519 do we want to default to true? idk

I think we should default this to false and let ENV variables take care of the feature configuration.